scripts/qa_crowbarsetup.sh

```diff
@@ -4451,11 +4465,7 @@ function onadmin_crowbarrestore()
             with_upgrade)
                 # restore after upgrade has different workflow (missing APIs) than
                 #   a restore from a backup of the same cloud release
-                local http_code=`curl -X POST -F "file=@/tmp/$btarball" -s -o upgrade-start.txt -w '%{http_code}' $crowbar_api/installer/upgrade/start.json`
```

Comment:
with the new commits now the unneeded authentication is removed again - so it should be the same as before